corehq/util/es/interface.py

```diff
@@ -122,6 +122,12 @@ def _fix_hits_in_results(self, results):
         for hit in hits:
             self._fix_hit(hit)
 
+        total = results['hits']['total']
+        # In ES7 total is a dict
+        if isinstance(total, dict):
+            results['hits']['total'] = total.get('value', 0)
```

Comment:
I'm curious why this is necessary? I thought we had code elsewhere to return `total.get("value", 0)` if `total` is a dict. Does this change mean that is no longer necessary? If yes, maybe that code should be removed?

Reply:
> Does this change mean that is no longer necessary?

Yes, this cleanup happens right after fetching the results, so the other ones are not necessary and I have removed them now.